src/dialects/abstract/query.d.ts

```diff
@@ -1,9 +1,7 @@
-import { Connection } from './connection-manager';
-import { QueryTypes } from '../../query-types';
-import { Model, ModelStatic, IncludeOptions } from '../../model';
-import { Sequelize } from '../../sequelize';
-
-type BindOrReplacements = { [key: string]: unknown } | unknown[];
```

Comment:
In this file some additional things are removed, such as the BindOrReplacement and replacementFuncType types and some TypeDoc is changed. Is that specific to the linting change?

Reply:
They became unused after the implementation of the new bind parameters :)